ert_shared/libres_facade.py

```diff
@@ -83,7 +84,7 @@ def run_path(self):
         return self._enkf_main.getModelConfig().getRunpathAsString()
 
     def load_from_forward_model(
-        self, case: str, realisations: BoolVector, iteration: int
+        self, case: str, realisations: List[bool], iteration: int
```

Comment:
I see that we use both spelling for `realization` and `realisation`. Maybe we should unify them at same point.

Reply:
Yes, that would be an issue of its own.